images/kubectl-build-deploy-dind/build-deploy-docker-compose.sh

```diff
@@ -88,7 +88,8 @@ do
     if kubectl --insecure-skip-tls-verify -n ${NAMESPACE} get service "$SERVICE_NAME" &> /dev/null; then
       SERVICE_TYPE="mariadb-single"
     # heck if this cluster supports the default one, if not we assume that this cluster is not capable of shared mariadbs and we use a mariadb-single
-    elif svcat --scope cluster get class $MARIADB_SHARED_DEFAULT_CLASS > /dev/null; then
+    # real basic check to see if the mariadbconsumer exists as a kind
+    elif kubectl --insecure-skip-tls-verify -n ${NAMESPACE} get mariadbconsumer &> /dev/null; then
```

Comment:
can you use this (adapt to your groups)
```
kubectl api-resources --no-headers --api-group=servicecatalog.k8s.io | grep serviceinstances -q
```
as with ^^ you can make sure that only your api-group is searched through, and not another one